tests/unit/test_deeprec_utils.py

```diff
@@ -0,0 +1,58 @@
+
+import pytest
+import os
+from reco_utils.recommender.deeprec.deeprec_utils import *
+from reco_utils.recommender.deeprec.IO.iterator import *
+from reco_utils.recommender.deeprec.IO.dkn_iterator import *
+import tensorflow as tf
+
+@pytest.fixture
+def resource_path():
+    return os.path.dirname(os.path.realpath(__file__))
+
+
+@pytest.mark.parametrize("must_exist_attributes", [
+    "FEATURE_COUNT", "data_format", "dim"
+])
+@pytest.mark.gpu
+@pytest.mark.deeprec
```

Comment:
same comment as before, `@pytest.mark.deeprec` is not needed

Reply:
I prefer to add this tag, because for my offline running, I only need to run some specific part of test cases, and offline run the entire test cases are too time consuming